lib/azure/armrest/model/storage_account.rb

```diff
@@ -249,6 +251,22 @@ def delete_blob(container, blob, key = nil, options = {})
         true
       end
 
+      def create_snapshot_blob(container, blob, key = nil)
+        key ||= properties.key1
+
+        url = File.join(properties.primary_endpoints.blob, container, blob)
+        url += "?comp=snapshot"
+
+        headers = build_headers(url, key, :blob, :verb => 'PUT')
+        headers['Content-Type'] = ''
+        response = RestClient.put(url, '', headers)
+
+        BlobSnapshot.new({
+          :Name => blob,
```

Comment:
Please align by `=>`
Keys here are strings and lower case (unless the raw API returns capitalized). So they are `'name'` and `'snapshot'`